doc/application_plugins/index.rst

```diff
@@ -165,17 +165,18 @@ With either setup, you can test the RADIUS setup using a command like this::
 Microsoft NPS server
 --------------------
 You can also use the Microsoft Network Protection Server with privacyIDEA.
-A full featured integration guide can be found at the NetKnights webpage
-[#NPS]_.
```

Comment:
What is wrong with the footnote references?

Reply:
Since the link is often an integral part of the text ("... see here.", "... get from"), footnotes are not appropriate (in my opinion).
I was thinking of adding footnotes additionally to the links (for people who don't like to click on links) .